src/main/java/org/spongepowered/common/mixin/core/world/gen/ChunkProviderServerMixin.java

```diff
@@ -92,6 +93,8 @@
     @Shadow protected abstract void saveChunkExtraData(Chunk chunkIn);
     @Shadow protected abstract void saveChunkData(Chunk chunkIn);
 
+    @Shadow public abstract boolean canSave();
```

Comment:
prefix with `shadow$` ( I get that this is in maintenance, and `api-8` branch is moving towards prefixing shadows always, but I'd like to keep it in line here as well).